plugins/repositories/core/src/main/javascript/app/components/add/add.css

```diff
@@ -1,12 +1,67 @@
-.add_container {
-  height: 440px;
-  overflow: hidden;
+.repository-manager-add-header {
+  font-family: OpenSansLight;
+  margin: 20px;
 }
 
-.pentaho_repo {
+.text-color-primary {
+  color: #005eaa;
+}
+
+.btn-color-primary {
+  color: white;
+  background-color: #005eaa;
+  border-color: #005eaa;
+}
+
+.btn-color-primary:hover {
+  color: white;
+  background-color: #0f2b5b;
+  border-color: #0f2b5b;
+}
+
+.btn-outline-color-secondary {
+  color: #005DA6;
+  background-color: #FFFFFF;
+  border: 1px solid #005DA6;
+}
+
+.btn-outline-color-secondary:hover {
+  color: #0F2B5B;
+  background-color: #FFFFFF;
+  border: 1px solid #0F2B5B;
+}
+
+#repository-manager-add-container {
```

Comment:
Do these comments need to stay? CSS are sent to the client, so they'll waste bandwidth.